src/js/text/main.html

```diff
@@ -59,16 +59,17 @@
 
   <div class='onward-signup'>
     <div class='onward-signup__container-title col'>
-      <div class='coming-soon'>Coming soon</div>
-      <div class='details'>Next up from the Guardian and Bertha’s groundbreaking documentary partnership</div>
+      <div class='coming-soon'>more documentaries</div>
+      <div class='details'>Watch more from the Guardian and Bertha’s groundbreaking documentary partnership</div>
     </div>
     <div class='onward-signup__contents col'>
       <div class='onward-signup__film-details col'>
         <span class='section-title'>Out on <strong data-sheet-attribute="nextDocDate"></strong></span>
         <span class='film-title' data-sheet-attribute="nextDocTitle"></span>
         <p class='film-synopsis' data-sheet-attribute="nextDocBody"></p>
-          <div class='docs--actions__trailer docs__shows-trailer' data-link-name="play-trailer">
-          <span class='docs--actions__trailer__duration'></span>
+        <a class='watch-now' href='https://www.theguardian.com/world/ng-interactive/2016/oct/14/desert-fire-the-world-cup-rebels-of-kurdistan-video?INTCMP=from_gun_nation_page'>Watch the documentary now</a>
+        <div class='docs--actions__trailer docs__shows-trailer trailer-button' data-link-name="play-trailer">
```

Comment:
from the screenshot, it doesn't look like there is a trailer on the page, is this still needed?